vowpalwabbit/core/src/array_parameters_dense.cc

```diff
@@ -89,6 +89,19 @@ void dense_parameters::clear_offset(const size_t offset, const size_t params_per
   }
 }
 
+void dense_parameters::adjust_weights_single_model(const size_t params_per_problem)
```

Comment:
Cool.

Reply:
Would be easy to change this to save model of any index.

Reply:
More comments here would be helpful.  How it works in conjunction with gd.save_load would be also useful for people looking at this code.  The reason for swap is subtle, a comment there would also be useful.

Reply:
Added a comment explaining the main operation of changing weights. I removed added a model index argument and removed to swap because that relied on the assumption that the other models were zero'ed out